sql/moz-fx-data-shared-prod/telemetry/active_users_aggregates/view.sql

```diff
@@ -59,10 +55,6 @@ SELECT
   wau,
   mau,
   new_profiles,
-  ad_clicks,
```

Comment:
@alekhyamoz The search_counts are corrected in https://github.com/mozilla/bigquery-etl/pull/4176
We need to merge this PR first, maintain the data in the view and update the data to backfill.